bin/xud

```diff
@@ -57,10 +57,15 @@ const { argv } = require('yargs')
       type: 'boolean',
       default: undefined,
     },
-    nosanitychecks: {
-      describe: 'Whether to disable order sanity checks',
+    nosanityswaps: {
+      describe: 'Whether to disable sanity swaps',
       type: 'boolean',
-      default: undefined,
+      default: true,
```

Comment:
One last thing, we should keep the default here as `undefined`, otherwise this will override the value in the config file. If we have `nosanityswaps = false` in the config file, and then run `xud` without passing any argument for `nosanityswaps`, `nosanityswaps` will wind up as true which is obviously undesirable. It's a quirk of yargs.